extensions/frontend-tiles/src/integration-test/IntegrationTest.test.ts

```diff
@@ -0,0 +1,59 @@
+import { IModelApp } from "@itwin/core-frontend";
+import { expect } from "chai";
+import { TestContext } from "./TestContext";
+import { obtainGraphicRepresentationUrl } from "../GraphicsProvider/GraphicRepresentationProvider";
+
+describe("MeshExportServiceIntegrationTest", () => {
+  let accessToken: any;
+  let testArgs: any;
+
+  before(async () => {
+    const testContext = await TestContext.instance();
+    accessToken = testContext.getAccessToken();
+
+    const iTwinId = process.env.TEST_ITWINID || "";
+    const iModelId = process.env.TEST_IMODELID || "";
+
+    testArgs = {
+      accessToken,
+      sessionId:"testSession",
+      dataSource: {
+        iTwinId,
+        id: iModelId,
+        changeId: "",
+        type: "IMODEL",
+      },
+      format: "IMDL",
+      urlPrefix: "qa-",
+    };
+
+    await IModelApp.startup();
+  });
+
+  after(async () => IModelApp.shutdown());
+
+  it ("should obtain a tileset with a tile version supported by the frontend", async () => {
```

Comment:
As discussed in the meeting with @arvindsvenkat - this test should probably request a new export.

Reply:
These changes have been made and are included in the most recent commit.